source/manual/github-setup.html.md

```diff
@@ -0,0 +1,33 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Set up your GitHub account
+layout: manual_layout
+section: Learning GOV.UK
+last_reviewed_on: 2019-07-09
+review_in: 6 months
+---
+
+1. Set up a [GitHub][] account.
+1. Ask somebody with access to add your GitHub username to the [user monitoring system][user-reviewer].
+1. Ask your tech lead to add you to the [alphagov organisation][alphagov]. You will have to be added to the [GOV.UK team][govuk-team] to get access to repos & CI.
+1. [Generate and register an SSH key pair][register-ssh-key] for your Mac for your GitHub account.
```

Comment:
Should we note that SSH keys [need to be 4096 bits (`ssh-keygen -b 4096`)](https://github.com/alphagov/govuk-puppet/blob/b588e4ade996e97b8975e69cb00800521fff4a48/modules/users/spec/classes/users_spec.rb#L57)?

Reply:
Updated!